### PR TITLE
remove ie11 from docs

### PR DESCRIPTION
**Pull request by HugoPct**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 1]
```diff
@@ -97,13 +97,13 @@ if (window._DATADOG_SYNTHETICS_BROWSER === undefined) {
...
 
-If you use the browser variable to identify Synthetic bots on Firefox and Internet Explorer 11, Datadog cannot guarantee that the browser variable is set before your code executes. 
+If you use the browser variable to identify Synthetic bots on Firefox, Datadog cannot guarantee that the browser variable is set before your code executes. 
 
 ## Cookies
 
 Cookies that are applied in your browser include `datadog-synthetics-public-id` and `datadog-synthetics-result-id`.
 
-These cookies are available for all steps in Firefox and Internet Explorer 11. For Edge and Google Chrome, these cookies are set only for the start URL.
+These cookies are available for all steps in Firefox. For Edge and Google Chrome, these cookies are set only for the start URL.
```
Comment (alai97) [inline content/en/synthetics/guide/identify_synthetics_bots.md:106]:
```suggestion
These cookies are available for all steps in Firefox. For Microsoft Edge and Google Chrome, these cookies are set only for the start URL.
```

**Review by alai97 — APPROVED** [review]

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ datadogRum.startSessionReplayRecording();
 
 Replace the Browser SDK URL `https://www.datadoghq-browser-agent.com/datadog-rum.js` with `https://www.datadoghq-browser-agent.com/datadog-rum-v4.js`. When `DD_RUM.init()` is called, the Session Replay recording does not start until `DD_RUM.startSessionReplayRecording()` is also called.
 
-The Session Replay recorder supports all browsers supported by the RUM Browser SDK with the exception of IE11. For more information, see the [browser support table][3].
+The Session Replay recorder supports all browsers supported by the RUM Browser SDK. For more information, see the [browser support table][3].
```
Comment (HugoPct) [inline content/en/real_user_monitoring/session_replay/_index.md:59]: we shouldn't change anything on the RUM side.
This is a synthetics thing only :)

Comment (HugoPct) [reply content/en/real_user_monitoring/session_replay/_index.md:59]: otherwise we are good to go

Comment (alai97) [reply content/en/real_user_monitoring/session_replay/_index.md:59]:
```suggestion
The Session Replay recorder supports all browsers supported by the RUM Browser SDK with the exception of IE11. For more information, see the [browser support table][3].
```

**Pull request merged**